test-suite/handwritten-src/cpp/extended_record.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include "extended_record_base.hpp"
+
+namespace testsuite {
+	
+	struct ExtendedRecord : public ExtendedRecordBase
+	{
+		using ExtendedRecordBase::ExtendedRecordBase;
+	};
+}
```

Comment:
Is there an extra null character here or something?  I'm not sure what GitHub is trying to point out.